Hippo.Core/Services/NotificationService.cs

```diff
@@ -348,6 +350,44 @@ await _emailService.SendEmail(new EmailModel
             }
         }
 
+        public async Task<bool> OrderNotificationTwoButton(SimpleNotificationModel simpleNotificationModel, Order order, string[] emails, string[] ccEmails = null)
+        {
+            try
+            {
+                var message = simpleNotificationModel.Paragraphs.FirstOrDefault();
```

Comment:
Why only the first paragraph for the text body? I would have joined them into a single string joined by `[CRLF][CRLF]`. Is it because order notifications won't ever have more than one paragraph?